saleor/graphql/account/types.py

```diff
@@ -452,9 +453,12 @@ class Group(CountableDjangoObjectType):
     permissions = graphene.List(
         PermissionDisplay, description="List of group permissions"
     )
+    user_can_manage = graphene.Boolean(
+        required=True, description="True if user has rights to manage group."
```

Comment:
To make it clear which user:
```suggestion
        required=True, description="True, if the currently authenticated user has rights to manage a group."
```